data-prepper-plugins/saas-source-plugins/saas-crawler/src/main/java/org/opensearch/dataprepper/plugins/source/saas/crawler/base/SaasSourceConfig.java

```diff
@@ -0,0 +1,9 @@
+package org.opensearch.dataprepper.plugins.source.saas.crawler.base;
+
+/**
+ * Marker interface to all the SAAS connectors configuration
+ */
+public interface SaasSourceConfig {
+
+    int DEFAULT_NUMBER_OF_WORKERS = 1;
```

Comment:
What is the motivation for having this here? Why would we have a default worker count of 1 for all SaaS connectors?

Reply:
Initial thinking was that we will take this input from the pipeline yaml configuration itself and let the user define it based on their own enterprise package they have with their service provider. Because, this decides the concurrency (pressure) we create on their service. Default value 1 meaning, that we start with least pressure possible but that also means the total data extraction takes a lot of time.

Reply:
For jira case, we are not planning to take this input from the customer. We will start with a reasonable default value suitable for Jira.